about.html

```diff
@@ -0,0 +1,114 @@
+              <li class="first-nav-list"><a href="#" class="first-nav-anchor">English</a></li>
+              <li class="first-nav-list"><a href="#" class="first-nav-anchor">My page</a></li>
+              <li class="first-nav-list"><a href="#" class="first-nav-anchor">Logout</a></li>
+          </ul>
+          </nav>
+      <nav class="second-nav none" id="nav">
+          <ul class="second-nav-logo">
+            <li><a href="./home.html"><i class="fa fa-first-order footer-icon"></i></a></li>
+              <li class="second-nav-logo-list">Global Tech 2021</li>
+          </ul>
+          <ul class="second-nav-ul">
+             <li><a href="./about.html" class="second-nav-anchor">About</a></li> 
+             <li><a href="#" class="second-nav-anchor">Program</a></li> 
+             <li><a href="#" class="second-nav-anchor">Join</a></li> 
+             <li><a href="#" class="second-nav-anchor">Sponsor</a></li> 
+             <li><a href="#" class="second-nav-anchor">News</a></li> 
+             <li><a href="#" class="second-nav-anchor last">Academics</a></li> 
+          </ul>
+      </nav>
+  </header>
+
+      <main>
+        <section class="main-section">
+            <h1 class="main-header m-top text-center w-100">"Hello! Technology World"</h1>
+            <h2 class="sub-header text-center w-100"> Web development & design courses 2021</h2>
+          <p class="about-paragraph-one paragraph text-center w-100">Lorem ipsum dolor sit amet, consectetur adipiscing elit.olor sit amet, consectetur adipiscing elit.
+               Fusce molestie pretium tristique. Quisque massa lorem, aliquam sed risus.Lorem ipsum dolor sit amet, 
+          </p>
+          <p class="venue text-center">Please contact 8us per Email for any further question about CC Global Summit 2021</p>
+          <p class="mail text-center">ccsumit2021@cckorea.org</p>
+          </section>
+
+          <section class="about-section">
+             <article class="about-article">
+                 <h2 class="about-header">
+                     Creative Commons Global Summit 2021 Logo
+                 </h2>
+                 <p class="about-section-paragraph text-center">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce molestie pretium tristique. 
+                    Quisque massa lorem, aliquam sed risus.Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                     Fusce molestie pretium tristique. Quisque massa lorem, aliquam sed risus.Lorem ipsum dolor sit amet, 
+               </p>
+                <ul class="about-paragraph">
+                    <li class="global-logo"><i class="fa fa-first-order footer-icon"></i> Global Tech 2021</li>
+                    <li class="location">14th - 17th OCT 2021 | Seoul, Korea</li>
+                </ul>
+             </article>
+             <article class="second-article">
+                 <h2 class="about-header">See the past CC Global Summits</h2>
```

Comment:
- [ ] Images in this section should be centered. Please remove the margin between them.
![Screenshot from 2021-07-01 20-24-44](https://user-images.githubusercontent.com/1642014/124165545-99872b00-daaa-11eb-96b6-878c07e63d9f.png)

Reply:
:x: This is not addressed yet, 

Yours: 
![Screenshot from 2021-07-01 22-14-56](https://user-images.githubusercontent.com/55918152/124184444-db6b9d80-dab9-11eb-9cb3-7b5141613627.png)

Kindly try to put them together in center

Reply:
Hello de-scholar, 
I have been able to make changes to this section by giving it a max-width of 720px and removed all margins
https://github.com/bimbolabuari/mv-html-capstone/blob/0eeb5bfafd46b7adf08ca9a0f227633c2423443c/styles.css#L624-L629

Here is a screenshot of how it is rendered on my desktop.
![Screenshot (293)](https://user-images.githubusercontent.com/66112760/124189422-94c97380-dab8-11eb-8044-3f2b8c14bfef.png)

Thanks, If you have any suggestions on how to improve this, I will gladly implement it 👍

Reply:
This issue is well addressed! :+1: